pyaptamer/utils/_fasta_to_aaseq.py

```diff
@@ -0,0 +1,62 @@
+__author__ = "satvshr"
+__all__ = ["fasta_to_aaseq"]
+
+import os
+from io import StringIO
+
+import pandas as pd
+from Bio import SeqIO
+
+from pyaptamer.utils._hf_to_dataset import hf_to_dataset
+
+
+def fasta_to_aaseq(fasta_path, return_df=False):
+    """
+    Extract sequences from a FASTA file, Hugging Face dataset, or URL.
+
+    Parameters
+    ----------
+    ``fasta_path`` : str or os.PathLike
+        Input source for FASTA sequences. Can be:
+
+          - Local file path (absolute or relative) located in
```

Comment:
is this all relative to the repo?

Note that users of the package will not have the repo cloned. Please change this so it is relative to, say, root, and/or that absolute paths not subpaths of the repo can be passed.

Reply:
Very rookie mistake on my end 😓